packages/nimble-components/src/rich-text/editor/index.ts

```diff
@@ -343,6 +343,7 @@ export class RichTextEditor extends FoundationElement implements ErrorPattern {
          */
         return new Editor({
             element: this.editor,
+            enableInputRules: [BulletList, OrderedList],
```

Comment:
It looks like you need `enablePasteRules` too. If I paste `**bold** and *italic*`, it still gets formatted.

We probably want test coverage for this too. If the editor behaves differently for paste and regular input, should we actually be running most of our tests in both modes?

Reply:
Thanks for the catch. I have disabled all the paste rules because the lists don't have any default paste rules and they have only input rules. Below are the example code references to show where they have enabled the paste rules:

Paste and input rules are enabled in italics: https://github.com/ueberdosis/tiptap/blob/b5cb36058b4d6331f8b7bbd4540b698fa97fce61/packages/extension-italic/src/italic.ts#L98C3-L98C16
Only input rule is enabled in a bullet list: https://github.com/ueberdosis/tiptap/blob/b5cb36058b4d6331f8b7bbd4540b698fa97fce61/packages/extension-bullet-list/src/bullet-list.ts#L71

I have written test cases for most of the markdown syntaxes in this PR. I also verified the code of the Tiptap editor where they have enabled paste rules for a few extensions like bold, italics, code, strike, and highlight, and not for other nodes and marks. Therefore, other pasting rules will behave as same as the regular inputs.